src/minimega/external.go

```diff
@@ -26,40 +26,37 @@ var (
 	MIN_KERNEL = []int{3, 18}
 )
 
-// externalProcessesLock mediates access to customExternalProcesses.
-var externalProcessesLock sync.Mutex
-
-// defaultExternalProcesses is the default mapping between a command and the
-// actual binary name. This should *never* be modified. If the user needs to
-// update customExternalProcesses.
-var defaultExternalProcesses = map[string]string{
-	"qemu":     "kvm",
-	"ip":       "ip",
-	"ovs":      "ovs-vsctl",
-	"dnsmasq":  "dnsmasq",
-	"kill":     "kill",
-	"dhcp":     "dhclient",
-	"openflow": "ovs-ofctl",
-	"mount":    "mount",
-	"umount":   "umount",
-	"mkdosfs":  "mkdosfs",
-	"qemu-nbd": "qemu-nbd",
-	"rm":       "rm",
-	"qemu-img": "qemu-img",
-	"cp":       "cp",
-	"taskset":  "taskset",
-	"lsmod":    "lsmod",
-	"ntfs-3g":  "ntfs-3g",
-	"scp":      "scp",
-	"ssh":      "ssh",
-	"hostname": "hostname",
-	"tc":       "tc",
+// externalDependencies contains all the external programs that minimega
+// invokes. We check for the existence of these on startup and on `check`.
+var externalDependencies = map[string]bool{
```

Comment:
Should we remove the dependencies on kill, mount (though I think there's a special need for the mount binary somewhere), hostname, cp, umount, and rm?